chapter_06_aggregate.asciidoc

```diff
@@ -672,6 +679,10 @@ def test_concurrent_updates_to_version_are_not_allowed(postgres_session_factory)
 ----
 ====
 
+// TODO (hynek) (re # type: List[Exception]) why don't you use the proper syntax
+// for variable type annotations
+// (HP) because its hella ugly.
```

Comment:
HOW IS A COMMENT LESS UGLY HAROLD

Reply:
because my syntax highlighter puts into a colour that immediately signals to me that it's unimportant and ignorable.